sbncode/CAFMaker/CAFMaker_module.cc

```diff
@@ -1368,6 +1414,8 @@ void CAFMaker::produce(art::Event& evt) noexcept {
   rec.hdr.det     = fDet;
   rec.hdr.fno     = fFileNumber;
   rec.hdr.pot     = fSubRunPOT;
+  if(fIsRealData)
+    rec.hdr.bnbinfo = bnb_info_store;
```

Comment:
This is duplicating all the bnbinfo into every event, right? I think we probably only want to write it once. Not sure if you should put an `&& fFirstInFile` here, or just a `.clear()` after you've copied it into the record.

Reply:
Yes, this is what we discussed before. I pushed this before resolving that. Will add that in too. I can try a `fFirstInFile` and see if that works that's really what we want